packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export const replaceFillAttribute = (element: Element, selectedColor: string): void => {
```

Comment:
looks the same as the functions added in AnchorWidget, can we reuse this function in there?

Reply:
done